source/read/count.txt

```diff
@@ -0,0 +1,145 @@
+.. uses tutorial.rst
+
+.. _pymongo-count:
+
+===============
+Count Documents
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: number, amount, estimation
+
+Overview
+---------
+
+In this guide, you can learn how to get an accurate and estimated count of the
+number of documents in your collection.
+
+Get an Accurate Count
+----------------------
+
+Use the ``count_documents()`` command to count the number of documents that are in a
+collection. To count the number of documents that match a specific search
+critera, pass the ``count_documents()`` method a dictionary that specifies a
+query.
+
+To learn more about specifying a query, see :ref:`pymongo-specify-query`.
+
+Count All Documents
+~~~~~~~~~~~~~~~~~~~
+
+To return a count of all documents in the collection, pass an empty dictionary to
+the ``count_documents()`` method, as shown in the following example:
+
+.. code-block:: python
+
+   collection = database["collection_name"]
```

Comment:
Q: do the code examples in this page need this line? Could you instead just show the actual method call?

Reply:
The intent was to make it clear that `collection` here is a variable for their collection rather than the method needing to be prepended with the word "collection" but I think that was probably over explaining a bit. Removed the line in favor of being more concise.